awswrangler/athena/_utils.py

```diff
@@ -71,6 +72,11 @@ def _start_query_execution(
         "OutputLocation": _get_s3_output(s3_output=s3_output, wg_config=wg_config, boto3_session=session)
     }
 
+    if client_request_token:
+        if len(client_request_token) < 32 or len(client_request_token) > 128:
+            raise exceptions.InvalidArgumentCombination("Invalid length for parameter client_request_token, valid min length: 32, valid max length: 128")
```

Comment:
Wouldn't this check already be handled by the boto client? If so, I think we can skip it to avoid unnecessary code